Compile fail on gentoo: undefined reference to `RealQuery::RealQuery(RealWindow*)'

On Gentoo linux with 4.9.95-gentoo kernel, I am not able to compile even after following #45. It failed for both x32 and x64.

The problem is that somehow RealQuery.cpp is not present in the build.ninja files for both x32 and x64. Any ideas?

64 Build log
```
[64/65] Linking target VK9-Library/d3d9.dll.
FAILED: VK9-Library/d3d9.dll 
/usr/bin/x86_64-w64-mingw32-g++  -o VK9-Library/d3d9.dll 'VK9-Library/VK9-Library@@d3d9@sha/BufferManager.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/C9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CBaseTexture9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CCubeTexture9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CDevice9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CIndexBuffer9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CPixelShader9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CQuery9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CRenderTargetSurface9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CResource9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CStateBlock9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CSurface9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CSwapChain9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CTexture9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CTypes.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CVertexBuffer9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CVertexDeclaration9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CVertexShader9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CVolume9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/CVolumeTexture9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/D3D9.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/dllmain.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/DrawContext.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/GarbageManager.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/Perf_CommandStreamManager.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/Perf_ProcessQueue.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/Perf_RenderManager.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/Perf_StateManager.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/RealDevice.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/RealIndexBuffer.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/RealInstance.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/RealSurface.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/RealSwapChain.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/RealTexture.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/RealVertexBuffer.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/RealWindow.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/ResourceContext.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/SamplerRequest.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/ShaderConverter.cpp.obj' 'VK9-Library/VK9-Library@@d3d9@sha/Utilities.cpp.obj' -Wl,--no-undefined -Wl,--as-needed -Wl,-O1 -shared ../../VK9-Library/d3d9.def -Wl,--start-group -Wl,--out-implib=VK9-Library/libd3d9.dll.a -pthread -lboost_system-mt-1_67 -lboost_filesystem-mt-1_67 -lboost_thread-mt-1_67 -lboost_program_options-mt-1_67 -lboost_log-mt-1_67 -lboost_log_setup-mt-1_67 -L/usr/x86_64-w64-mingw32/var/opt/x86_64-w64-mingw32/vulkan/lib -lvulkan-1 -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 -Wl,--end-group -static -static-libgcc -static-libstdc++ 
VK9-Library/VK9-Library@@d3d9@sha/Perf_StateManager.cpp.obj:/usr/lib/gcc/x86_64-w64-mingw32/7.3.0/include/g++-v7/ext/aligned_buffer.h:99: undefined reference to `RealQuery::RealQuery(RealWindow*)'
VK9-Library/VK9-Library@@d3d9@sha/Perf_StateManager.cpp.obj:/usr/lib/gcc/x86_64-w64-mingw32/7.3.0/include/g++-v7/ext/new_allocator.h:140: undefined reference to `RealQuery::~RealQuery()'
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```

--- a/VK9-Library/meson.build
+++ b/VK9-Library/meson.build
@@ -31,6 +31,7 @@ d3d9_src = [
   'RealDevice.cpp',
   'RealIndexBuffer.cpp',
   'RealInstance.cpp',
+  'RealQuery.cpp',
   'RealSurface.cpp',
   'RealSwapChain.cpp',
   'RealTexture.cpp',

